src/components/content/heroes/aurora/aurora.tsx

```diff
@@ -59,6 +73,66 @@ const AuroraContainer = styled.div`
   pointer-events: none;
 `;
 
+const AuroraRainyFlareBackground1 = styled.div`
+  right: -25%;
+  top: -20%;
+  width: 1094px;
+  height: 1094px;
+  border-radius: 1094px;
+  background: linear-gradient(180deg, #150b28 0%, rgba(35, 33, 131, 0.15) 100%);
+  filter: blur(125px);
+  position: absolute;
+`;
+
+const AuroraRainyFlareBackground2 = styled.div`
```

Comment:
The term "Background" can a little bit misleading here. It seems like this component is a background for the aurora hero. But they are just different colored flares, right?

What do you think about replacing "Background" with "Color" to avoid confusion?